### ISSUE TITLE
ASSERTION VIOLATION

### ISSUE THREAD

**Issue by Nosferican** [Z3Prover/z3#4728]
```
ASSERTION VIOLATION
File: /workspace/srcdir/z3/src/api/api_ast.cpp
Line: 906
UNREACHABLE CODE WAS REACHED.
4.8.8.0
```
Error during Slurm job in cluster.
> Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, haswell)

**Comment by NikolajBjorner** [comment]
looks like there is a stale header file compared to the dll.

**Comment by Nosferican** [comment]
What's the proposed solution? Is that being addressed as part of #4699?

**Comment by NikolajBjorner** [comment]
> It is very common that users have header files and libraries that are out of sync and it can be tricky to debug
because the way header files and libraries are picked up depends on path variables.
> You are not the first and will not be the last posing such an issue.
